test/metabase/upload_test.clj

```diff
@@ -472,21 +472,47 @@
           (when grant?
             (perms/revoke-data-perms! group-id db-id)))))))
 
+(defn do-with-uploads-allowed
+  "Set uploads-enabled to true, and uses an admin user, run the thunk"
+  [thunk]
+  (mt/with-temporary-setting-values [uploads-enabled true]
+    (mt/with-current-user (mt/user->id :crowberto)
+      (thunk))))
+
+(defmacro with-uploads-allowed [& body]
+  `(do-with-uploads-allowed (fn [] ~@body)))
+
+(defn- do-with-upload-table! [t f]
```

Comment:
```suggestion
(defn- do-with-upload-table! [table thunk]
```
nit: short var is hard to read IMO

Reply:
Done ed612a658c32be08a5e7264b3cf0ecafe3a13ce3